Cds 92419 (#6700)
harness/developer-hub: docs/platform/git-experience/configure-git-experience-for-harness-entities.md

  - You add the PAT to Harness as a Text Secret and it is encrypted using a Harness Secret Manager.
  - Your Git Personal Access Token is stored in your Harness secret and is a private key to which only you have access. This secret cannot be accessed or referenced by any other user.

   Make sure you configure SSO for your GitHub token when enabling Git provider access via SSO.

  - The PAT must have the following scope:
      - GitHub:![](./static/configure-git-experience-for-harness-entities-35.png)
      - Bitbucket:![](./static/configure-git-experience-for-harness-entities-36.png)
  - To enable Git Experience for your resources, make sure that you have Create/Edit permissions for them.​​

:::info note

> If you are using [Harness Code Repository](/docs/code-repository/) as your Git Repository, you will not require a Git Connector or a PAT.

Reviewer: ```suggestion
If you are using [Harness Code Repository](/docs/code-repository/) as your Git Repository, you won't need a Git Connector or a PAT.
```

For this, you must specify a Harness Git Connector, a repo, branch details, and a file path.

:::info note

> If you are using [Harness Code Repository](/docs/code-repository/) as your Git Repository, you will not require a Git Connector.

Reviewer: ```suggestion
If you are using [Harness Code Repository](/docs/code-repository/) as your Git Repository, you won't need a Git Connector.
```

![](./static/configure-git-experience-for-harness-entities-39.png)

> Here you can choose to use [**Harness Code Repository**](/docs/code-repository/) or a third party provider. If you choose Harness Code Repository, you will not need to configure a **Git Connector** and can skip to selecting your **Repository** below.

Reviewer: ```suggestion
Here you can choose to use [**Harness Code Repository**](/docs/code-repository/) or a third-party provider. If you choose Harness Code Repository, you won't need to configure a **Git Connector** and can skip to selecting your **Repository** below.
```

> Otherwise, in **Git Connector**, select or create a Git Connector to the repo for your Project. For steps, see [Code Repo Connectors](../connectors/code-repositories/connect-to-code-repo.md).

Reviewer: ```suggestion
Otherwise, in **Git Connector**, select or create a Git Connector for your project's repo. For steps, go to [Code Repo Connectors](../connectors/code-repositories/connect-to-code-repo.md).
```
